README.md

```diff
@@ -43,10 +43,14 @@ How does the package use scverse data structures (please describe in a few sente
 -   [ ] The package uses automated software tests and runs them via continuous integration (CI)
 -   [ ] The package provides API documentation via a website or README
 -   [ ] The package uses scverse datastructures where appropriate (i.e. AnnData, MuData or SpatialData and their modality-specific extensions)
--   [ ] I am the author or maintainer of the tool and agree on listing the package on the scverse website
+-   [ ] I am a author or maintainer of the tool and agree on listing the package on the scverse website
 
 ### Recommended
 
--   [ ] I would like this package to be announced on scverse channels (zulip, discourse, twitter) [Include twitter handles you want us to tag]
+-   [ ] Please announce this package on scverse communication channels (zulip, discourse, twitter)
+-   [ ] Please tag the author(s) these announcements. Handles (e.g. `@scverse_team`) to include are:
+    -   Twitter:
+    -   Zulip:
+    -   Discourse:
```

Comment:
```suggestion
    -   Discourse:
    -   Mastodon: 
```

@mikelkou, are we using this now?

Reply:
Good point with the handles. 
Yes, we are using Mastodon (specifically Fosstodon) and LinkedIn.